test/hotspot/gtest/logging/test_asynclog.cpp

```diff
@@ -73,17 +73,23 @@ LOG_LEVEL_LIST
   // We would observe missing loglines if we interleaved buffers.
   // Emit all logs between constructor and destructor of BufferUpdater.
   void test_asynclog_drop_messages() {
-    const size_t sz = 2000;
+    const size_t sz = AsyncLogBufferSize / 2;
+    const char* str = "a lot of log...";
+    const size_t str_size = strlen(str);
 
     // shrink async buffer.
-    AsyncLogWriter::BufferUpdater saver(1024);
+    //AsyncLogWriter::BufferUpdater saver(1024);
```

Comment:
You have some comments that no longer apply once you comment this out. It is also unclear why you are commenting it out and not just deleting it.

Reply:
Hi, the TL;DR is that that was a mistake.

The long reason is: I wanted to try skipping the buffer and so I committed and pushed it to make it accessible on my build computer, but I forgot that it's up for a PR. That's actually the reason for all the comments.